ui/admin/tests/e2e/README.md

```diff
@@ -98,7 +100,8 @@ Enos needs some configuration variables to run the scenario successfully. [See t
 - `enos_user`: The user name to use for Boundary.
 - `tfc_api_token`: you need to provide the previously created token in Terraform cloud, there is no shared token within Boundary team.
 - `local_boundary_dir`: The directory that contains Boundary binary.
-- `local_boundary_ui_dir`: The directory that contains the copy of boundary-ui you want to use for UI tests.
+- `local_boundary_src_dir`: The directory that contains the copy of boundary you want to use for e2e tests
```

Comment:
Is this needed? I thought this was for testing the boundary BE e2e tests

Reply:
I got an error when I didn't add these during the setup mentioned in [step](https://github.com/hashicorp/boundary-ui/tree/main/ui/admin/tests/e2e#setup-enos) in the [config](https://github.com/hashicorp/boundary/blob/main/enos/enos.vars.hcl)

Reply:
Did it work for you without these changes? 🤔

Reply:
I only needed the UI parameter, though I only tried running our UI tests

Reply:
`local_boundary_src_dir` is not need for UI e2e testing purposes. I tested and works no problem without `local_boundary_src_dir`

Reply:
thanks for confirming, I will remove the field